### ISSUE TITLE
LICENSE missing in PyPI tarball

### ISSUE THREAD

**Issue by dotlambda** [jaraco/keyring#314]
Installation fails because `LICENSE` is referenced in `setup.cfg`.

**Comment by jaraco** [comment]
That's surprising, because keyring uses setuptools_scm, which pulls the file manifest from Git.

**Comment by jaraco** [comment]
But indeed, that seems to be the case.

```
$ curl -s https://files.pythonhosted.org/packages/d6/bb/aae5ea03a60f15b000a36b8822d401ef15589dcf7d93b566007e442246c5/keyring-12.0.0.tar.gz -o - | tar xz --include LICENSE
tar: LICENSE: Not found in archive
tar: Error exit delayed from previous errors.
```

**Comment by jaraco** [comment]
Looking for the [release build for the latest release](https://travis-ci.org/jaraco/keyring/jobs/355523900), I can see setuptools_scm 1.16.1 was used.

**Comment by jaraco** [comment]
Looking at [the changelog](https://github.com/pypa/setuptools_scm/blob/master/CHANGELOG.rst#v1170), I'm pretty sure 1.16.1 was buggy about files and the issue has been fixed in 1.17. I'll cut a new release.

**Comment by jaraco** [comment]
I'm assuming this is fixed. I can't wait for the machines to catch up to verify the fix. If you don't find the LICENCE file in the 12.0.1 release, ping me here.
